src/VisualStudio/Core/Def/Implementation/ProjectSystem/LinkedFileUtilities.cs

```diff
@@ -152,5 +152,23 @@ private string GetActiveIntellisenseProjectContextInternal(IVsHierarchy hierarch
                 ? intellisenseProjectName as string
                 : null;
         }
+
+        public static bool TryGetItemIdInSharedHierarchy(IVsHierarchy hierarchy, uint itemId, IVsHierarchy sharedHierarchy, out uint itemIdInSharedHierarchy)
+        {
+            string fullPath;
```

Comment:
I guess the distinction only matters if someone passes in a project itemId here, but that's not what this helper is designed for, so it's probably fine as "filePath"